_scss/_perldoc.scss

```
@@ -86,3 +86,8 @@
 .highlight .vi { color: #00688B } /* Name.Variable.Instance */
 .highlight .vm { color: #00688B } /* Name.Variable.Magic */
 .highlight .il { color: #B452CD } /* Literal.Number.Integer.Long */
+
+@media print {
+.highlight .k { color: #8B008B !important } /* Keyword */
+.highlight .s { color: #CD5555 !important } /* Literal.String */
+}
```

Looks like this is missing a newline at the end

```suggestion
}

```

---

Do you recall why the `!important` was needed? (trying to avoid those if possible)

Also wondering if it would be better to move this to `_night-mode.scss`

---

> Looks like this is missing a newline at the end

I now added the suggested newline character to the changes I'm going to commit with the next update.

> Do you recall why the !important was needed? (trying to avoid those if possible)

Sure, in the original `_night-mode.scss`, someone used `!important` quite abundant:

```scss
    blockquote {
        ...
        &.warning, &.warning p:first-child {
            color: $warning-color-night !important;
        }
        &.important {
            border-left-color: #f5ac45 !important;
        }
        &.important, &.important p:first-child {
            color: $important-color-night !important;
        }
    }
    ...
    pre {
        color: #f3f3f3;
        background-color: #0a121b !important;
        border: 1px solid $black;
    }
```

So, in order to be able to override these settings, I was forced to apply the `!important` modifier, too. I didn't want to take the effort to refactor the existing SCSS files.

> Also wondering if it would be better to move this to `_night-mode.scss`

That's a matter of style (no pun intended 😉). I chose the `_perldoc.scss` location so someone in the future may be taking the task to update the corresponding syntax highlight colors. He/she would then recognize the dark mode versions and alter them accordingly.

If you want me to move them to `_night-mode.scss` that'd be perfectly alright with me. I'm looking forward to your valuable suggestion.